std/string.d

```diff
@@ -3889,6 +3916,7 @@ auto rightJustifier(Range)(Range r, size_t width, dchar fillChar = ' ')
 
             void popFront()
             {
+                import std.range.primitives : popFront;
```

Comment:
As [I tried to explain previously](https://github.com/dlang/phobos/pull/4365#discussion_r64879054), just because a function accepts a range, it's not always obvious which implementation of `front`/`popFront`/`empty` it's using.  The compiler figures that out for you based on the type of range, so you also have to read other parts of the function and its constraints in order to know that. These imports are a documentation shortcut, since the functions from `std.range.primitives` are implemented for built-in arrays, so if I see this import, I immediately know that's the one being used.